agreement/player_permutation_test.go

```diff
@@ -86,7 +98,7 @@ func getPlayerPermutation(t *testing.T, n int) (plyr *player, pMachine ioAutomat
 		pM.Children[r].Children[p].ProposalTracker.Staging = pV
 		pM.Children[r].Children[p].ProposalTrackerContract.SawOneVote = true
 		pM.Children[r].Children[p].update(0)
-	case 5: // already reached cert threshold
+	case playerSameRound_ReachedCertThreshold: // already reached cert threshold
 		plyr, pMachine, helper = setupP(t, r, p, soft)
```

Comment:
Related, should this be `setupP(t, r, p, cert)`?

Reply:
The more common scenario is probably when its in the `cert` step, but it's also possible for this to happen in the `soft` scenario. Feel free to change it to `cert`

Reply:
OK, AFAICT in the `player.handle()` and `player.handleMessageEvent()` paths exercised by these tests, the only place the `p.Step` value is checked is in a clause `p.Step <= cert` so it doesn't matter.